LeapModules.uproject

```diff
@@ -1,23 +1,23 @@
-{
-	"FileVersion": 3,
-	"EngineAssociation": "4.25",
-	"Category": "",
-	"Description": "",
-	"Modules": [
-		{
-			"Name": "LeapModules",
-			"Type": "Runtime",
-			"LoadingPhase": "Default"
-		}
-	],
-	"Plugins": [
-		{
-			"Name": "LeapMotion",
-			"Enabled": true
-		},
-		{
-			"Name": "LiveLink",
-			"Enabled": true
-		}
-	]
+{
+	"FileVersion": 3,
+	"EngineAssociation": "{BE17D662-4BFF-BE3B-2383-6290E4D20230}",

```

Comment:
I don't think we want this change?

Reply:
I think it's ok. This will try and auto associate with 4.26.2 if installed. UE nags you to 'update the project file' and then adds the file version bit. Fine to remove it though, does it cause issues?

Reply:
This is what I see opening this project:
![image](https://user-images.githubusercontent.com/53301524/120307707-3481bf00-c2cb-11eb-9a1b-9b9e5fa025f1.png)